mm/slib_env.c

```diff
@@ -0,0 +1,484 @@
+/*
+ * Library Slab Allocator (SLIB)
+ *
+ * Copyright (c) 2015 INRIA, Hajime Tazaki
```

Comment:
this line (Copyright (c)...) should describe the copyright holder, apparently not INRIA and myself.